src/main/webapp/WEB-INF/web.xml

```diff
@@ -66,4 +66,14 @@
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
 
+  <servlet>
+    <servlet-name>ProfileServlet</servlet-name>
+    <servlet-class>codeu.controller.ProfileServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>ProfileServlet</servlet-name>
+    <url-pattern>/users/*</url-pattern>
```

Comment:
This is cool.  Will this make it so that you can go directly to someone's profile page just by going to the address .../users/their_name ?